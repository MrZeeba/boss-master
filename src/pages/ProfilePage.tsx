import React from 'react';
import { Text, View } from 'react-native';
import { globalColours } from '../globalColours';

export default function ProfilePage() {
  return (
    <View
      style={{
        flexDirection: 'row',
        height: 400,
        padding: 20,
      }}>
      <View style={{ backgroundColor: globalColours.primary, flex: 0.3 }}>
        <Text>#72705B</Text>
      </View>
      <View style={{ backgroundColor: globalColours.highlight, flex: 0.3 }}>
        <Text>#E8C547</Text>
      </View>
      <View style={{ backgroundColor: globalColours.secondary, flex: 0.3 }}>
        <Text>#50C878</Text>
      </View>
    </View>
  );
}
