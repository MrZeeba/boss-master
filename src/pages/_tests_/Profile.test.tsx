import React from 'react';
import renderer from 'react-test-renderer';

import ProfilePage from '../ProfilePage';

describe('<View />', () => {
  it('has 3 children', () => {
    const tree = renderer.create(<ProfilePage />).toJSON();
    expect(tree.children.length).toBe(3);
  });
});

it('renders correctly', () => {
  const tree = renderer.create(<ProfilePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
