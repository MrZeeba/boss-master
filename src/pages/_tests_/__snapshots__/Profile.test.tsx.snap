// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "flexDirection": "row",
      "height": 100,
      "padding": 20,
    }
  }
>
  <View
    style={
      {
        "backgroundColor": "blue",
        "flex": 0.3,
      }
    }
  />
  <View
    style={
      {
        "backgroundColor": "red",
        "flex": 0.5,
      }
    }
  />
  <Text>
    Hello World!
  </Text>
</View>
`;
